includes/definitions/arris-cm.yaml

```diff
@@ -0,0 +1,10 @@
+os: arris-cm
```

Comment:
Definitely the right os name to use? would arris-docsis not fit better? Do they have a name for the firmware?

Reply:
I started out as arris-docsis, then I saw the name of their MIB was arris-cm...  Perhaps I should include the mibs even though they are not used?

Reply:
Up to you, I used to tell people not to when I could see they weren't used but they can be useful later on.

Let me know what you decide or merge when you feel it's ready.